nautobot/core/choices.py

```diff
@@ -11,6 +11,27 @@ def __iter__(cls):
         choices = getattr(cls, "CHOICES", ())
         return iter(choices)
 
+    def __getattribute__(self, attr):
```

Comment:
This is not currently used, but it could help in getting the label of a choice field by adding the .label to the choice field, e.g 
`InterfaceStatusChoices.Status_Active` ==> `active`
`InterfaceStatusChoices.Status_Active.label` ==> `Active`.